pkg/crc/cache/cache.go

```diff
@@ -51,13 +46,13 @@ func (c *Cache) GetBinaryName() string {
 	return c.binaryName
 }
 
-/* getVersionGeneric runs the cached binary with 'args', and assumes the version string
+/* GetVersionGeneric runs the cached binary with 'args', and assumes the version string
  * was output on stdout's first line with this format:
  * something: <version>
  *
  * It returns <version> as a string
  */
-func getVersionGeneric(binaryPath string, args ...string) (string, error) {
+func GetVersionGeneric(binaryPath string, args ...string) (string, error) {
```

Comment:
This does not seem needed.

Reply:
yes the linter was complaining the function was not used for Windows. nolint annotations didnt work. It was the easiest way to make it pass.

Reply:
```
-func getVersionGeneric(executablePath string, args ...string) (string, error) {
+func getVersionGeneric(executablePath string, args ...string) (string, error) { //nolint:deadcode,unused
```
works for me with `golangci-lint` 1.31 and `make cross-lint`

Reply:
thanks it works! I forgot the comma